Sources/TuistLoader/Models+ManifestMappers/Project+ManifestMapper.swift

```diff
@@ -66,6 +69,22 @@ extension TuistCore.Project {
                 xcodeProjPath: xcodeProjPath,
                 name: name,
                 organizationName: organizationName,
+                developmentRegion: developmentRegion,
+                settings: settings,
+                filesGroup: filesGroup,
+                targets: targets,
+                packages: packages,
+                schemes: schemes,
+                additionalFiles: additionalFiles)
+    }
+
+    func replacing(developmentRegion: String?) -> TuistCore.Project {
+        Project(path: path,
```

Comment:
I think we can leverage the fact that `TuistCore.Project` is no longer a class and do a simpler version of this:
```swift
var project = self
project.developmentRegion = developmentRegion
return project
```

Reply:
In Project+ManifestMapper there are multiple `replacing` methods and none of them are being used throughout the project. I added this method to match the others. I can also make this change to the other `replacing` methods but could you please shortly explain why these methods even are in the project when they are not used?

Reply:
Good question 🙂 

`replacing` methods have been used back when `TuistCore.Project` was a class - to make the functions side effects-free, we needed to return a new class which is what `replacing` was for. But now it is a value type and these methods are not necessary. If they are not used anywhere, feel free to delete those, too